src/communication/network_interface_cl.c

```diff
@@ -1686,7 +1686,7 @@ heap_destroy (const HFID * hfid)
 
   THREAD_ENTRY *thread_p = enter_server ();
 
-  success = xheap_destroy (thread_p, hfid, NULL);
+  success = xheap_destroy (thread_p, hfid);
```

Comment:
When is the hfid removed from the cache in this case?
- previous:
  - the heap_delete_hfid_from_cache() is called by the vacuum_rv_notify_dropped_file() if the class_oid in not null
- now:
  - the heap_delete_hfid_from_cache() is called by the xheap_destroy_newly_created() only
 
Doesn't it need to remove from the cache?